<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProduktyDialog</class>
 <widget class="QDialog" name="ProduktyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add product</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QSpinBox" name="MagazynEdit">
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="StanMagazynuLable">
     <property name="text">
      <string>Capacity</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QDoubleSpinBox" name="PojemnoscEdit">
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.250000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="KategoriaBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="MagazynLable">
     <property name="text">
      <string>Magazine count</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="NazwaEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="CenaJednostkowaLable">
     <property name="text">
      <string>Price per one</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="IDKategoriiLable">
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDoubleSpinBox" name="CenaEdit">
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>2.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="NazwaProduktuLable">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProduktyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProduktyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NazwaEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ProduktyDialog</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CenaEdit</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>ProduktyDialog</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>accept()</slot>
  <slot>onTextChanged()</slot>
 </slots>
</ui>
